airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py

```diff
@@ -324,13 +358,13 @@ def request_params(
             }
 
         stream_date = stream_state.get(self.cursor_field)
-        start_date = next_page_token or stream_date or self.start_date
```

Comment:
We don't need `next_page_token` here, because `next_page_token` is relative URL for the next chunk of results.